### PR TITLE
Kanchenjunga: Version 2.001 added



### PR DESCRIPTION
**Pull request by emmamarichal**
Taken from the upstream repo https://github.com/silnrsi/font-kanchenjunga/ at commit https://github.com/silnrsi/font-kanchenjunga//commit/19a3efac0c2ca42a1b28a53c74ca8f3eb6c12ca7.
## PR Checklist:

- [ ] `minisite_url` definition in the METADATA.pb file for commissioned projects
- [ ] `primary_script` definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target
- [ ] `subsets` definitions in the METADATA.pb reflect the actual subsets and languages present in the font files (in alphabetic order). For **CJK fonts**, only include one of the following subsets `chinese-hongkong`, `chinese-simplified`, `chinese-traditional`, `korean`, `japanese`.
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [ ] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

**Comment by simoncozens** [comment]
This is new to Unicode so we will need to set up lang for it. Subsets should be in place but good to confirm with eng. Very exciting to have a Kirat Rai font!

File: ofl/kanchenjunga/article/ARTICLE.en_us.html
Comment (vv-monsalve) [inline ofl/kanchenjunga/article/ARTICLE.en_us.html:1]: Article/Description text is needed.

**Review by vv-monsalve — DISMISSED** [review]

**Comment by vv-monsalve** [comment]
@emmamarichal The `name ID 3 Unique font identifier` is changing from `SIL International: Harmattan Regular 2023` to `SIL Global: Harmattan Regular 2024`. Is this OK? Have we seen this already in previous upgrades, and did this not bring any issue?

**Comment by emmamarichal** [comment]
@vv-monsalve here is the progress of the primary script issue: https://github.com/googlefonts/lang/issues/180#issuecomment-2447892859

It's almost ready, we can hold on this PR until the lang update is released!

I also added the description!

**Comment by emmamarichal** [comment]
@vv-monsalve I just added the primary script tag to the `metadata.pb`. @simoncozens is working a the new lang release. Once it's merged to the gf repo, we will wait then for the eng team feedback before merging this one. I already broke the dev-server because the primary script wasn't totally ready 🙃

**Comment by vv-monsalve** [comment]
@emmamarichal Thanks for the update. Just ping me when it's ready for review.

**Comment by emmamarichal** [comment]
@vv-monsalve the lang PR have been merged: https://github.com/google/fonts/pull/8451
So I think you can go ahead with the review, but we should ask to the eng team before merging it!

**Comment by vv-monsalve** [comment]
@emmamarichal

Taking the sample text introduced for the [Krai language](https://github.com/googlefonts/lang/blob/main/Lib/gflanguages/data/languages/bap_Krai.textproto) the font is showing tofu. Please take a look to identify if something is missing in the font or if the sample text needs to be adjusted.

Screenshot 2024-11-07 at 14 14 21

**Comment by simoncozens** [comment]
My fault, bad sample text.

**Comment by simoncozens** [comment]
(https://github.com/googlefonts/lang/pull/189)

**Comment by vv-monsalve** [comment]
> ([googlefonts/lang#189](https://github.com/googlefonts/lang/pull/189))

No tofu after the last sample.

Screenshot 2024-11-13 at 20 24 57

@simoncozens, should we wait for that change to be pulled into this repo or land elsewhere before merging this PR?

File: ofl/kanchenjunga/article/ARTICLE.en_us.html
Comment (vv-monsalve) [reply ofl/kanchenjunga/article/ARTICLE.en_us.html]: @emmamarichal I saw you marked this as resolved, but the font still needs the Article.

Comment (emmamarichal) [reply ofl/kanchenjunga/article/ARTICLE.en_us.html]: Ah? I added it two weeks ago, it doesn't show up on your side?

Comment (vv-monsalve) [reply ofl/kanchenjunga/article/ARTICLE.en_us.html]: It does now!

**Comment by simoncozens** [comment]
Looks like we need a lang release + subtree pull

**Comment by simoncozens** [comment]
Lang PR here: https://github.com/google/fonts/pull/8509

**Comment by emmamarichal** [comment]
Nyshadh confirmed: 
_confirmed that the new script is on the dev servers. Above PR is safe to merge._

**Comment by emmamarichal** [comment]
Update: script isn't displayed in dev-sandbox

https://github.com/google/fonts/pull/8509 Should unblock it, but it doesn't work yet.

Screenshot 2024-12-06 at 11 11 27

@nyshadhr9, do you know why the script isn't displayed?

**Comment by emmamarichal** [comment]
Kanchenjunga is working great in dev-sandbox! But not in sandbox.

I see the PR that sloved it is marked as 'in sandbox': https://github.com/google/fonts/pull/9221 
@garretrieger, Kanchenjunga should work in sandbox too if I understand correctly. Can you check that?

cc @m4rc1e 


Screenshot 2025-04-10 at 10 10 48

**Comment by garretrieger** [comment]
Sorry I updated that list a bit prematurely, there was a failure in the push earlier. However that's resolved now and it should be on sandbox now: https://fonts-sandbox.corp.google.com/specimen/Kanchenjunga?query=Kanchenjunga

**Comment by emmamarichal** [comment]
@garretrieger thank you very much! I don't see it yet here: https://fonts.sandbox.google.com/specimen/Kanchenjunga but I guess it will be pushed soon!

**Pull request merged**
